content/messaging/agent-chat-api/v3.3/index.mdx

```diff
@@ -118,6 +118,33 @@ The **File** event informs about an uploaded file.
 </Code>
 </Section>
 
+## Form
+
+<Section>
+<Text>
+
+The **Form** event contains data from a form (sent during agent inactivity for example).
```

Comment:
I don't like this phrasing - it's almost the same as for `Filled form` while these 2 events are actually opposite (empty form vs. filled form)